tests/request.spec.coffee

```diff
@@ -286,3 +253,49 @@ describe 'Request:', ->
 							foo: 'bar'
 					.get('body')
 					m.chai.expect(promise).to.eventually.become(foo: 'bar')
+
+		describe 'given an endpoint that fails the first two times', ->
+
+			beforeEach ->
+				requestsSeen = 0
+				fetchMock.get 'https://example.com/initially-failing', ->
```

Comment:
Generally even for smaller tweaks we'll open up a PR, that way it will still get some eyes on it in case some small mistake was added by accident - after all it can happen to any of us, especially as the small fixes also tend to be the ones that get paid the least attention to.  Actually for a lot of our repos it's impossible to push directly to master and changes have to go via a PR and won't let you merge until the tests have passed as well, it'd be good to set that up on even more repos imo